recipe/build.sh

```diff
@@ -1,24 +1,42 @@
 #!/bin/bash
 
-export LDFLAGS="$LDFLAGS $(pkg-config --libs ncurses)"
-export CPPFLAGS="$CPPFLAGS $(pkg-config --cflags-only-I ncurses)"
-export CFLAGS="$CFLAGS $(pkg-config --cflags-only-I ncurses)"
+# Prevent running ldconfig when cross-compiling.
+if [[ "${BUILD}" != "${HOST}" ]]; then
+  echo "#!/usr/bin/env bash" > ldconfig
+  chmod +x ldconfig
+  export PATH=${PWD}:$PATH
+fi
+
+
+export CPPFLAGS="${CPPFLAGS} -DSQLITE_ENABLE_COLUMN_METADATA=1 \
+                             -DSQLITE_ENABLE_UNLOCK_NOTIFY \
+                             -DSQLITE_ENABLE_DBSTAT_VTAB=1 \
+                             -DSQLITE_ENABLE_FTS3_TOKENIZER=1 \
+                             -DSQLITE_SECURE_DELETE \
+                             -DSQLITE_MAX_VARIABLE_NUMBER=250000 \
+                             -DSQLITE_MAX_EXPR_DEPTH=10000 \
+                             -DSQLITE_ENABLE_GEOPOLY \
+                             -DSQLITE_ENABLE_JSON1 \
+                             -DSQLITE_ENABLE_RTREE=1"
```

Comment:
@snorfalorpagus do you know if there are any simple tests we can do to test RTREE abd GEOPOLY support here?